log4j-core/pom.xml

```diff
@@ -44,7 +44,6 @@
       <!-- Annotations only -->
       org.jspecify.*;resolution:=optional,
       <!-- External optional dependencies -->
-      org.apache.commons.compress.*;resolution:=optional,
```

Comment:
1. Shouldn't we have `org.jspecify;transitive=false` in `bnd-extra-module-options`?
2. Why didn't we have `org.apache.commons.compress.*;transitive=false` in `bnd-extra-module-options`?

Reply:
1. Maybe, if it appears as `transitive`, I'll fix it when merging #2930
2. BND adds `transitive` only if a dependency type appears in the signature of a public method or extends/implements clauses of a public class/interface. This is not the case of Commons Compress.